applications/OptimizationApplication/custom_utilities/response/linear_strain_energy_response_utils.h

```diff
@@ -0,0 +1,83 @@
+//    |  /           |
```

Comment:
There is a scenario that is not covered in this implementation and in the implementation on the master. What if we are interested in the strain energy of a submodel part of the analysis model part? In that case, we would need to solve for adjoints and so on. Should we cover this now, or will it be implemented later if needed?"

Reply:
Let's implement it when it is required?